rosidl_generator_cpp/resource/msg__traits.hpp.em

```diff
@@ -28,12 +28,15 @@ cpp_namespace = '%s::%s::' % (spec.base_type.pkg_name, subfolder)
 #include <stdint.h>
 #include <type_traits>
 
-namespace rosidl_generator_traits
+namespace rosidl_generator_cpp
```

Comment:
Can you please clarify why you changed this namespace? Won't this break any existing code using it?

Reply:
I changed it because in issue #258 it was mentioned that "...which uses the rosidl_generator_traits namespace for all traits, but I think that might be better changed to rosidl_generator_cpp or something similar which is rosidl and cpp specific."

I am reverting the namespace changes as of now.